components/landing.tsx

```diff
@@ -32,7 +31,9 @@ const Header2 = styled(H2)`
 
 const Container = styled.div`
```

Comment:
@AKCodez - can you confirm this component (Container) isn't used on other pages outside of the Featured one?